sensa/__version__.py

```diff
@@ -0,0 +1,11 @@
+"""
+Metadata for sensa-python
+"""
+__title__ = "sensa-python"
+__description__ = "Python module for the TecnoTree Sensa Platform"
+__url__ = "https://github.com/CognitiveScale/sensa-python"
+__version__ = "6.4.0"
+__author__ = "TecnoTree"
+__author_email__ = "TBD?"
+__license__ = "Apache 2.0"
+__copyright__ = "TecnoTree LLC. All Rights Reserved."
```

Comment:
We need to make sure this `copyright` field matches the copyright statement included throughout the code - see the top of `setup.py`, `sensa/client.py`, etc. (this is one reason why I'm not fan of having the License string duplicated in every module).